Mathlib/GroupTheory/Subsemigroup/Center.lean

```diff
@@ -56,6 +56,10 @@ structure IsMulCentral [Mul M] (z : M) : Prop where
   /-- associative property for right multiplication -/
   right_assoc (a b : M) : (a * b) * z = a * (b * z)
 
+attribute [mk_iff isMulCentral_iff] IsMulCentral
+attribute [mk_iff isAddCentral_iff] IsAddCentral
+attribute [to_additive existing] isMulCentral_iff
```

Comment:
Annoyingly, this makes the lemmas have too many explicit arguments. Mind proving them by hand?

Reply:
I'd rather you just file an issue to complain and we leave a TODO. There's no point having mk_iff at all if we never use it because it generates things badly.

Reply:
#9129

Reply:
I added a TODO